units/en/_toctree.yml

```
@@ -92,6 +92,22 @@
     title: Quick Quiz 2
   - local: unit2/llama-index/conclusion
     title: Conclusion
+- title: Unit 2.3 LangGraph
+  sections:
+  - local: unit2/langgraph/introduction
+    title: Introduction to LangGraph
+  - local: unit2/langgraph/when_to_use_langgraph
+    title: When to use LangGraph ?
```

The title differs from the section title
```suggestion
    title: What is LangGraph?
```